Modelica/Electrical/Machines/Utilities/LogicalDelay.mo

```diff
@@ -0,0 +1,30 @@
+within Modelica.Electrical.Machines.Utilities;
+block LogicalDelay "Delay boolean signal"
+  extends Blocks.Icons.BooleanBlock;
+  parameter SIunits.Time delayTime(final min=0)=0 "Time delay";
+  Blocks.Interfaces.BooleanInput u
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Blocks.Interfaces.BooleanOutput y1
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Blocks.Interfaces.BooleanOutput y2
+  annotation (Placement(transformation(extent={{100,-70},{120,-50}})));
+protected
+  Boolean not_u=not u;
```

Comment:
It looks unnecessary to introduce `not_u` when you could just write `not u` in the two places this is used.

Reply:
resolved by 8d1bf4e3f..b3435eb49